content/general/presence.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: Presence
+section: general
+index: 27
+---
+
+Presence allows you to know the connection status of a client or a device. Apart from letting you know who is online/offline at a particular point in time, you can also subscribe to presence updates( Note: This is only available with Ably's "Realtime Library":/realtime ). This means that Ably will send a message to all it's presence subscribers whenever a client on the same presence channel that they are subscribed to, has left or a new client has joined.
+
+<a href="/images/diagrams/presence.png" target="_blank">
+  <img src="/images/diagrams/presence.png" style="width: 100%" alt="Presence representation">
```

Comment:
What is this diagram representing?

Reply:
Just a representative diagram of clients and/or devices being online or offline. Any suggestions?